optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/single/constraintmatchtotalbestscore/ConstraintMatchTotalBestScoreSingleStatistic.java

```diff
@@ -205,7 +205,7 @@ public void writeGraphFiles(BenchmarkReport benchmarkReport) {
             }
             plot.setDataset(seriesCollection);
             JFreeChart chart = new JFreeChart(
-                    singleBenchmarkResult.getName() + " constraint match total best score diff level " + scoreLevelIndex + " statistic",
+                    solverProblemBenchmarkResult + " constraint match total best score diff level " + scoreLevelIndex + " statistic",
```

Comment:
The lacking of the .getName() call means it will use to toString() method instead.
As this shows up in the UI (the benchmark report), it should still use the getName() method. toString() is primary for debugging and logs. In many cases this is hair splitting indeed, as they are often the same.